algorithms/C/tree/max-heap.c

```diff
@@ -0,0 +1,109 @@
+#include <stdio.h>
```

Comment:
use [code beautifier](https://codebeautify.org/cpp-formatter-beautifier) to fix indent as well as perfect space between letter